website/content/docs/agent-and-proxy/proxy/caching/index.mdx

```diff
@@ -7,21 +7,17 @@ description: |-
   newly created tokens.
 ---
 
-# Vault proxy caching
+# Vault Proxy caching
 
-Vault Proxy Caching allows client-side caching of responses containing newly
+Vault Proxy caching allows client-side caching of responses containing newly
```

Comment:
clarify this is for dynamic secrets?

Reply:
The following text should be clear enough, no?:
`responses containing newly created tokens and responses containing leased secrets generated off of these newly created tokens`

This describes the subset of dynamic secrets that we do support caching

Reply:
I think just calling out explicitly that this is dynamic secrets makes things more clear, but this is fine for now.